File: FlexServ Backend/SpringBoot/AdminService/src/main/java/com/flexserv/dto/response/BookingResponse.java
package com.flexserv.dto.response;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.LocalTime;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class BookingResponse {

    private Long id;

    private Long customerId;

    private String customerName;

    private String customerEmail;

    private Long providerId;

    private String providerCompanyName;

    private Long serviceId;

    private String serviceName;

    private LocalDate bookingDate;

    private LocalTime bookingTime;

    private String status;

    private BigDecimal totalPrice;

    private Boolean isActive;

    private LocalDateTime createdAt;
}

File: FlexServ Backend/DotNet/AdminService/Dtos/BookingResponse.cs
using System;

namespace AdminService.Dtos
{
    public class BookingResponse
    {
        public long Id { get; set; }
        public long? CustomerId { get; set; }
        public string? CustomerName { get; set; }
        public string? CustomerEmail { get; set; }
        public long? ProviderId { get; set; }
        public string? ProviderCompanyName { get; set; }
        public long? ServiceId { get; set; }
        public string? ServiceName { get; set; }
        public DateOnly? BookingDate { get; set; }
        public TimeSpan? BookingTime { get; set; }
        public string? Status { get; set; }
        public decimal? TotalPrice { get; set; }
        public DateTime? CreatedAt { get; set; }

        public BookingResponse() { }

        public BookingResponse(long id, long? customerId, string? customerName, string? customerEmail,
            long? providerId, string? providerCompanyName, long? serviceId, string? serviceName,
            DateOnly? bookingDate, TimeSpan? bookingTime, string? status, decimal? totalPrice, DateTime? createdAt)
        {
            Id = id;
            CustomerId = customerId;
            CustomerName = customerName;
            CustomerEmail = customerEmail;
            ProviderId = providerId;
            ProviderCompanyName = providerCompanyName;
            ServiceId = serviceId;
            ServiceName = serviceName;
            BookingDate = bookingDate;
            BookingTime = bookingTime;
            Status = status;
            TotalPrice = totalPrice;
            CreatedAt = createdAt;
        }
    }
}
